test/unit/decay.c

```diff
@@ -2,10 +2,63 @@
 
 #include "jemalloc/internal/decay.h"
 
-/*
- * Honestly, this is mostly a stub for now.  Eventually, we should beef up
- * testing here.
- */
+TEST_BEGIN(test_decay_init) {
+	decay_t decay;
+	memset(&decay, 0, sizeof(decay));
+
+	nstime_t curtime;
+	nstime_init(&curtime, 0);
+
+	ssize_t decay_ms = 1000;
+	assert_true(decay_ms_valid(decay_ms), "");
+
+	expect_false(decay_init(&decay, &curtime, decay_ms),
+	    "Should be able to initialize");
+	expect_zd_eq(decay_ms_read(&decay), decay_ms,
+	    "Correctly initializes decay ms");
```

Comment:
Very nitty, but since this is also in part for codebase acclimation: A little more in keeping with our other tests is to say what went wrong rather than what the expectation is testing. I.e. here it might be "decay ms initialized incorrectly".

Reply:
Got it, will make sure all `expect`s follow this rule.